migration/codec.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package migration;
+
+import "github.com/iov-one/weave/codec.proto";
+
+// Schema declares the maxiumum supported schema version for a package.
+message Schema {
+  weave.Header header = 1;
```

Comment:
Haha... so funny. I was confused between `header.schema` and `version`.... Then realized the first is the schema of the model in Migrations package, second is the schema version in the referenced package.

Looks good

Reply:
I think adding some sample init code for this would be nice demo.

Reply:
A handler will happen soon :wink: